modules/quick-start-content-guidelines.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-content-guidelines_{context}"]
+= Quick start content guidelines
+
+[id="quick-start-content-guidelines-card-copy_{context}"]
+== Card copy
+
+You can customize the title and description on a quick start card, but you cannot customize the status.
+
+* Keep your description to 1 to 2  sentences.
+* Start with a verb and communicate the user's goal. Correct example:
++
+----
+Create a serverless application.
+----
+
+[id="quick-start-content-guidelines-introduction_{context}"]
+== Introduction
+
+After clicking a quick start card, a side panel slides in that introduces the quick start and lists the tasks within it.
+
+* Make your introduction content clear, concise, informative, and friendly.
+* State the outcome of the quick start. A user should understand the purpose of the quick start before they begin.
+* Give action to the user, not the quick start.
+** *Correct example*:
++
+----
+In this quick start, you will deploy a sample application to {product-title}.
+----
+** *Incorrect example*:
++
+----
+This quick start shows you how to deploy a sample application to {product-title}.
+----
+* The introduction should be a maximum of 4 to 5 sentences, depending on how complex the feature is. A long introduction can overwhelm the user.
+* List the quick start tasks after the introduction content, and start each task with a verb. Do not specify the number of tasks because the copy would need to be updated every time a task is added or removed.
+** *Correct example*:
++
+----
+Tasks to complete: Create a serverless application; Connect an event source; Force a new revision
+----
+** *Incorrect example*:
++
+----
+You will complete these 3 tasks: Creating a serverless application; Connecting an event source; Forcing a new revision
+----
+
+[id="quick-start-content-guidelines-task-steps_{context}"]
+== Task steps
+
+Once the user clicks *Start*, a series of steps appears that they must perform to complete the quick start.
+
+Follow these general guidelines when writing task steps, in addition to the linked style guides at the beginning of this section:
+
+* Use "Click" for buttons and labels. Use "Select" for checkboxes, radio buttons, and drop-down menus.
+* Use "Click" instead of "Click on"
+** *Correct example*:
```

Comment:
If you do the `.Correct example` title styling mentioned earlier, then do the same in this module.